test/Microsoft.TestPlatform.CrossPlatEngine.UnitTests/Client/Parallel/ParallelOperationManagerTests.cs

```diff
@@ -57,9 +58,11 @@ public void OperationManagerShouldRunOnlyMaximumParallelLevelOfWorkInParallelEve
             // and pass on the current manager that is done.
             parallelOperationManager.RunNextWork(manager);
         };
+        Func<SampleManager, SampleHandler, SampleWorkload, Task> initializeWorkload = (_, _, _) =>
+            Task.Run(() => System.Threading.Thread.Sleep(100));
```

Comment:
Why are you not using `Task.Delay` here instead? Question is true for the other instances of this file.

Reply:
@nohwnd do you see anything wrong with using `Task.Delay` here ?

Reply:
No. As long as you await it.

Reply:
Fixed, awaited a `Task.Delay(100)` instead. I just saw this code was test code actually. The reason why `Thread.Sleep(100)` was used was probably to stay consistent with the rest of the tests which use this.